src/containers/NdlaFilm/components/DropdownSearch.jsx

```diff
@@ -12,42 +12,63 @@ import { AsyncDropdown } from '../../../components/Dropdown';
 import { searchResources } from '../../../modules/search/searchApi';
 import config from '../../../config';
 
-const queryResources = async input => {
-  const query = {
-    page: 1,
-    subjects: 'urn:subject:20',
-    'context-types': config.ndlaFilmArticleType,
-    sort: '-relevance',
-    'page-size': 10,
-    query: input,
+const DropdownSearch = ({
+  selectedElements,
+  placeholder,
+  onChange,
+  subjectId,
+  clearInputField,
+  onClick,
+}) => {
+  const queryResources = async input => {
+    let query = {
+      page: 1,
+      subjects: `urn:subject:${subjectId || '20'}`,
+      sort: '-relevance',
+      'page-size': 10,
+      query: input,
+    };
+    subjectId
+      ? (query = { ...query })
+      : (query = {
+          ...query,
+          'context-types': config.ndlaFilmArticleType,
+        });
```

Comment:
DropdownSearch brukes i ndla film og i subjectpageeditor, og for ndla film så trenger den en ekstra context-type (?), så lot være å sende inn subjectid for ndla film for å bruke den som indikator til å endre queryen, men vet ikke om det er litt dumt?

Reply:
Nei, det funker jo dette. Synes det ser greut ut